### PR TITLE
Test: Set skeletondirectory when running tests

### LINKED ISSUE #2568: Test: Set skeletondirectory when running tests
We already set skeletondirectory automagically on core. It'd be great to have it set here as well.

### PR DESCRIPTION
**Pull request by skshetry**
## Description

This PR sets `skeletondirectory` when running tests. Sadly, `client` is not ready when`BeforeAll` runs, so I have to run this in `Before`.

## Related Issue

- Fixes #2568

## Motivation and Context

Make it easier to run tests for everyone, and one less thing for me to remember

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by skshetry** [comment]
Since this does not affect CI, I request all reviewers to make sure this works when they checked out this branch locally. :)

**Comment by skshetry** [comment]
@dpakach, `core` sets whatever it needs to write on the core, and if someone tries to run acceptance tests, it will swiftly shift to the required skeleton directory, so, that won't be much problem (except for polluting folders for new users). But, I don't have strong opinion towards any, but would prefer to not reset the values at the end for simplicity.

File: tests/acceptance/setup-local.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
 import { setDefaultTimeout, After, Before } from 'cucumber'
-import { createSession, closeSession, startWebDriver, stopWebDriver } from 'nightwatch-api'
+import { createSession, closeSession, startWebDriver, stopWebDriver, client } from 'nightwatch-api'
+import join from 'join-path'
```
Comment (individual-it) [inline tests/acceptance/setup-local.js]: shouldn't we use `join` from `helpers/path`? see #2535

Comment (skshetry) [reply tests/acceptance/setup-local.js]: I was just lazy to import from `./helpers/path`, as the behavior is same. Fixing it.

File: tests/acceptance/helpers/config.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  const newConfig = await getConfigs()
+
+  const appConfig = _.get(newConfig, 'apps')
+  const systemConfig = _.get(newConfig, 'system')
+
+  const initialAppConfig = _.get(config, 'apps')
+  const initialSysConfig = _.get(config, 'system')
+
+  await rollbackSystemConfigs(
+    difference(systemConfig, initialSysConfig),
```
Comment (individual-it) [inline tests/acceptance/helpers/config.js]: feels like I would give the function both configs and do the diff inside the rollback function

File: tests/acceptance/helpers/config.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
+const fetch = require('node-fetch')
+const httpHelper = require('./httpHelper')
+const occHelper = require('./occHelper')
+const { join } = require('./path')
+const { difference } = require('./objects')
+const _ = require('lodash')
+const pLimit = require('p-limit')
+
+// run 10 promises at once at max
+const limit = pLimit(10)
```
Comment (individual-it) [inline tests/acceptance/helpers/config.js]: why that?

Comment (skshetry) [reply tests/acceptance/helpers/config.js]: After your previous review, I made changing all those configurations concurrent. Just tried to future-proof and not overwhelm our computers. I know 10 is already a lot.

**Pull request merged**